userguide/payment/userguide_payment.adoc

```diff
@@ -14,7 +14,7 @@ We will assume:
 * You have gone through https://docs.killbill.io/latest/what_is_kill_bill.html[What is Kill Bill] and are familiar with Kill Bill
 * You have gone through the https://docs.killbill.io/latest/getting_started.html[Getting started] tutorial and have Kill Bill, Kaui and the database setup and running
 * You have a tenant configured with API key `bob` and API secret `lazar`
-* You have http://curl.haxx.se/[cURL] installed. This is only to be able to run the setup steps and examples from this documentation. In practice, your application will use one of our https://killbill.github.io/slate/[client libraries]
+* You have http://curl.haxx.se/[cURL] installed. This is only to be able to run the setup steps and examples from this documentation. In practice, your application will use one of our https://killbill.github.io/slate/[client libraries]. If you are on Windows, we recommend that you use https://git-scm.com/download/win[Git Bash] to run the cURL commands.
```

Comment:
```suggestion
* You have https://curl.haxx.se/[cURL] installed. This is only to be able to run the setup steps and examples from this documentation. In practice, your application will use one of our https://killbill.github.io/slate/[client libraries]. If you are on Windows, we recommend that you use https://git-scm.com/download/win[Git Bash] to run the cURL commands.
```

Reply:
okay, I've fixed this here, but we may need to fix other docs that use "http".

Reply:
I did a search of all the docs (not APIs) and thought I'd tracked them all down, but obviously I missed this one!